### PR TITLE
CEDS-1543 Added validation for additional fiscal references

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/services/cache/ExportItem.scala [hunk 1]
```diff
@@ -43,8 +43,13 @@ case class ExportItem(
     fiscalInformation.exists(_.onwardSupplyRelief == FiscalInformation.AllowedFiscalInformationAnswers.yes)
 
   def isCompleted: Boolean =
-    procedureCodes.isDefined && fiscalInformation.isDefined && itemType.isDefined &&
+    procedureCodes.isDefined && isFiscalInformationCompleted && itemType.isDefined &&
       packageInformation.nonEmpty && commodityMeasure.isDefined && additionalInformation.isDefined
+
+  private def isFiscalInformationCompleted: Boolean =
```
Comment (bambuchaAdm) [inline app/services/cache/ExportItem.scala]: can we move this to fiscal information object ?

File: app/services/cache/ExportItem.scala [hunk 2]
```diff
@@ -43,8 +43,13 @@ case class ExportItem(
...
+    if (fiscalInformation.isDefined && fiscalInformation.map(_.onwardSupplyRelief) == Some(yes))
```
Comment (bambuchaAdm) [inline app/services/cache/ExportItem.scala]: fiscalInformation.exists(_.isDefined)

Comment (PatrykRudnicki) [reply app/services/cache/ExportItem.scala]: Changed implementation

File: app/services/cache/ExportItem.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply app/services/cache/ExportItem.scala]: this method based on information in fiscal information and additional fiscal information. Decision about completed is connected only from whole item perspective. I think this is better place for this method

**Pull request merged**